code/src/components/ThoughtList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Moment from 'moment';
```

Comment:
Not everything imported from outer packages (from node_modules) should be written in uppercase - only components should.

So, for example this is component and should be imported in uppercase :

```jsx
import Datepicker from 'react-datepicker';
```

But this is NOT component but a function and therefore should be imported in lowercase :

```jsx
import moment from 'moment';
```